pkg/ndm-grpc/protos/ndm.proto

```diff
@@ -0,0 +1,101 @@
+syntax = "proto3";
+
+package ndm;
+
+// Might need add github.com/openebs/node-disk-manager and change the path of this file to avoid depreciation
+option go_package="ndm";
+
+
+// This service can return generic information 
+service Info {
+    // FindVersion Method gets the version information of the NDM components installed
+    rpc FindVersion (Null) returns (VersionInfo);
+
+  }
+
+// This service returns status of ISCSI
+service ISCSI{
+    rpc Status(Null) returns (ISCSIStatus);
+  }
+
+service Node{
+      
+  //  FindNodeName method is used find the name of the node on which NDM is running on 
+    rpc Name(Null) returns (NodeName) ;
+  
+  // ListBlockDevices method is used to return all the block devices found in hierarchy by NDM
+  rpc ListBlockDevices(Null) returns (BlockDevices);
+  
+  // ListBlockDeviceDetails method is used to find SMART info about a block device.
+  // Only the name field is required for input 
+  rpc ListBlockDeviceDetails(BlockDevice) returns (BlockDeviceDetails); 
+
+  // Set hugePages on this node
+  rpc SetHugepages(Hugepages) returns (HugepagesResult);
+
+  // Gets the number of huge pages set on the node
+  rpc GetHugepages(Null) returns (Hugepages);
+  
+  // Rescan syncs etcd and NDM's local state
+  rpc Rescan(Null) returns (Message);
+}
+
+message Message{
+  string msg = 1;
+}
+
+message Hugepages{
+  int32 pages = 1;
```

Comment:
I think we have both `2Mi` and `1Gi` pages. The name should specify which page we are returning.

Reply:
Will a comment do? Since we're only using 2Mi pages?